doc/configure

```diff
@@ -118,7 +131,7 @@ def man_to_html(man_name, man_out, builds):
 
 
 def convert_man_dir_to_man(
-        src_dir, dst_dir, rule, html_mans, builds, extra_inputs=None):
+        src_dir, dst_dir, rule, html_mans, builds, html, extra_inputs=None):
```

Comment:
`html` is not a very good name for a boolean value

Reply:
`generate_html` or `gen_html`?

Reply:
Thanks Kareem. I went with `gen_html`.